examples/FilterExample.js

```diff
@@ -55,7 +55,7 @@ class FilterExample extends React.Component {
   _onFilterChange(e) {
     if (!e.target.value) {
       this.setState({
-        filteredDataList: this._dataList,
+        filteredDataList: this._dataList
```

Comment:
But this one (arguments) is different. Keeping trailing commas in objects and arrays wont break the build, i guess.